tools/renumber-migrations

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
```

Comment:
From what I've read in the Zulip coding guide, the first line of every python doc should be
`from __future__ import absolute_import`
It might be superfluous at the moment, but relevant for future edits.